specification/elastic/resource-manager/Microsoft.Elastic/preview/2022-05-05-preview/elastic.json

```diff
@@ -0,0 +1,2302 @@
+      "post": {
+        "tags": [
+          "TrafficFilter"
+        ],
+        "summary": "Create and Associate private link traffic filter for the given deployment.",
+        "operationId": "createAndAssociatePLFilter_Create",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "name",
+            "description": "Name of the traffic filter",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "privateEndpointGuid",
+            "description": "Guid of the private endpoint",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "privateEndpointName",
+            "description": "Name of the private endpoint",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "createAndAssociatePLFilter_Create": {
+            "$ref": "./examples/PrivateLinkTrafficFilters_Create.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/associateTrafficFilter": {
+      "post": {
+        "tags": [
+          "TrafficFilter"
+        ],
+        "summary": "Associate traffic filter for the given deployment.",
+        "operationId": "AssociateTrafficFilter_Associate",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "rulesetId",
+            "description": "Ruleset Id of the filter",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success"
+          },
```

Comment:
Need to include the final 200 response as well

Reply:
Added